Added TID for execsnoop & open

afek854 opened the pull request:
Hey, i'm from the kubescape team. 
We would like to have `TID` as an additional enrichment to those events. 
Thanks!

mauriciovasquezbernal reviewed (approved):
LGTM. I'll kick the CI and merge when it's green. Thanks for the contribution!

mauriciovasquezbernal commented:
@afek854 before kicking the CI, would you mind squashing all commits into one please?

mauriciovasquezbernal commented:
@afek854 I think you're missing a `make ebpf-objects` invokation: https://github.com/inspektor-gadget/inspektor-gadget/actions/runs/11707111312/job/32608767572?pr=3658

mauriciovasquezbernal commented:
It seems it's missing an update to the unit tests too: https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/gadgets/trace/exec/tracer/tracer.go

afek854 commented:
> It seems it's missing an update to the unit tests too: https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/gadgets/trace/exec/tracer/tracer.go

I updated the tests, but I noticed that some unittests fails on things I did not modify. 
Can you be more specific please?

mauriciovasquezbernal commented:
> I updated the tests, but I noticed that some unittests fails on things I did not modify. Can you be more specific please?

I'm not sure why but the `TestExecTracer/event_has_correct_paths` unit test seems to be broken after this PR. 

https://github.com/inspektor-gadget/inspektor-gadget/actions/runs/11763968090/job/32779521206?pr=3658

afek854 commented:
> > I updated the tests, but I noticed that some unittests fails on things I did not modify. Can you be more specific please?
> 
> I'm not sure why but the `TestExecTracer/event_has_correct_paths` unit test seems to be broken after this PR.
> 
> https://github.com/inspektor-gadget/inspektor-gadget/actions/runs/11763968090/job/32779521206?pr=3658

I fixed it, my bad
